### ISSUE TITLE
Build error with 7.2.0

### ISSUE THREAD

**Issue by mat813** [anttiviljami/openapi-client-axios#157]
I don't exactly understand what happened, but with 7.2.0, I get this build error:

```
$ tsc
node_modules/openapi-client-axios/client.d.ts(135,13): error TS2411: Property 'link' of type 'AxiosHeaders | undefined' is not assignable to 'string' index type 'string | number | boolean | string[] | AxiosHeaders'.
node_modules/openapi-client-axios/client.d.ts(136,13): error TS2411: Property 'head' of type 'AxiosHeaders | undefined' is not assignable to 'string' index type 'string | number | boolean | string[] | AxiosHeaders'.
node_modules/openapi-client-axios/client.d.ts(137,13): error TS2411: Property 'options' of type 'AxiosHeaders | undefined' is not assignable to 'string' index type 'string | number | boolean | string[] | AxiosHeaders'.
node_modules/openapi-client-axios/client.d.ts(138,13): error TS2411: Property 'get' of type 'AxiosHeaders | undefined' is not assignable to 'string' index type 'string | number | boolean | string[] | AxiosHeaders'.
node_modules/openapi-client-axios/client.d.ts(139,13): error TS2411: Property 'delete' of type 'AxiosHeaders | undefined' is not assignable to 'string' index type 'string | number | boolean | string[] | AxiosHeaders'.
node_modules/openapi-client-axios/client.d.ts(140,13): error TS2411: Property 'post' of type 'AxiosHeaders | undefined' is not assignable to 'string' index type 'string | number | boolean | string[] | AxiosHeaders'.
node_modules/openapi-client-axios/client.d.ts(141,13): error TS2411: Property 'put' of type 'AxiosHeaders | undefined' is not assignable to 'string' index type 'string | number | boolean | string[] | AxiosHeaders'.
node_modules/openapi-client-axios/client.d.ts(142,13): error TS2411: Property 'patch' of type 'AxiosHeaders | undefined' is not assignable to 'string' index type 'string | number | boolean | string[] | AxiosHeaders'.
node_modules/openapi-client-axios/client.d.ts(143,13): error TS2411: Property 'purge' of type 'AxiosHeaders | undefined' is not assignable to 'string' index type 'string | number | boolean | string[] | AxiosHeaders'.
node_modules/openapi-client-axios/client.d.ts(144,13): error TS2411: Property 'unlink' of type 'AxiosHeaders | undefined' is not assignable to 'string' index type 'string | number | boolean | string[] | AxiosHeaders'.
node_modules/openapi-client-axios/client.d.ts(145,13): error TS2411: Property 'common' of type 'AxiosHeaders | undefined' is not assignable to 'string' index type 'string | number | boolean | string[] | AxiosHeaders'.
error Command failed with exit code 2.
```

I don't think I am the problem, but I don't really get it.

**Comment by dumdiedum** [comment]
I'm also getting same error when building. I could localize problem within ./node_modules/openapi-client-axios/client.d.ts:

![grafik](https://github.com/anttiviljami/openapi-client-axios/assets/29304583/67e327fd-0f91-4fd4-a2cd-70e73cf53205)

I can "fix" it locally when adding "| undefined" at line 134:
![grafik](https://github.com/anttiviljami/openapi-client-axios/assets/29304583/43d406db-94f2-4ef5-98e9-4ce3496da5a3)

Maybe this will help fixing this, @anttiviljami?

**Comment by anttiviljami** [comment]
Hi @dumdiedum @mat813, thanks for reporting this. Couldn't find a way to reproduce this but hopefully adding the correct return type fixes the issue for you 🙏

Published as part of `openapi-client-axios@7.3.1`

**Comment by mat813** [comment]
It did, thank you.
